tests/casperesults.xml

```diff
@@ -0,0 +1 @@
dress line 3 text input field in the lightbox as expected" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0"/><testcase name="Found address postcode text input field in the lightbox as expected" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Found 'Save details' button in the lightbox as expected" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0"/><testcase name="Found cancel button in the lightbox as expected" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Correct validation message shown for blank title" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.228"/><testcase name="Correct validation message shown for too long first name" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Correct validation message shown for too long last name" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Correct validation message shown for too long address line 1" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Correct validation message shown for too long address line 2" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Correct validation message shown for too long address line 3" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Certificate provider's name is displayed on landing page" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.549"/><testcase name="Found view/edit link for changing certificate provider details" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0"/><testcase name="[Save and continue] button on certificate provider page is visible and pointing to people-to-notify page as expected" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Lightbox for editing certificate provider's details is loaded and displayed as expected" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.197"/><testcase name="Found certificate provider form as expected" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Title dropdown is correctly generated" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="First name is correctly loaded" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0"/><testcase name="Last name is correctly loaded" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Address1 is correctly loaded" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Address2 is correctly loaded" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0"/><testcase name="Address3 is correctly loaded" classname="tests/05-CreatePfLpa/10-certificate-provider" time="0.001"/><testcase name="Postcode is correctly loaded" classname="t
```

Comment:
This file probably shouldn't be commited.

Reply:
removed